projects/demo/used-icons.ts

```diff
@@ -54,6 +54,8 @@ export const TUI_USED_ICONS = [
     '@tui.message-square',
     '@tui.info-filled',
     '@tui.info',
+    '@tui.badge-russian-ruble',
```

Comment:
Let's try to use icons we already use to simplify proprietary demo

Reply:
fixed